tests/apache/hive/sensors/test_hive_partition.py

```diff
@@ -15,63 +15,54 @@
 TEST_METASTORE_CONN_ID = "test_metastore_default"
 
 
-@pytest.fixture()
-def context():
-    """Creates an empty context."""
-    context = {}
-    yield context
+class TestHivePartitionSensorAsync:
+    def test_hive_partition_sensor_async(self, context):
```

Comment:
is context be reused from `airflow_utils.py`?

Reply:
No, this is coming from conftest.py